sherlock/internal/api/sherlock/git_commits_v3_upsert.go

```diff
@@ -54,15 +53,14 @@ func gitCommitsV3Upsert(ctx *gin.Context) {
 	var timeSince *uint
 
 	if len(previous) > 0 {
-		timeSince = utils.PointerTo(uint(body.CommittedAt.Sub(previous[0].CommittedAt).Seconds()))
+		timeSince = utils.PointerTo(uint(body.CreatedAt.Sub(previous[0].CreatedAt).Seconds()))
```

Comment:
**suggestion:** Based on my comment above, the body won't have a `.CreatedAt`. Instead I think we should just use `time.Now()` (not leaving it as a GitHub suggestion because you'd need to import `"time"` too, which GoLand will do if you write it yourself)